.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Ticket
+[Ticket number](link to ticket)
+
+### Description
+Please put here the intent of your pull request.
+
+### Dependencies
+List the other pull requests that should be merged before/along this one.
+
+### Checklist
+- [ ] Referenced a ticket in the PR title and in the corresponding section
```

Comment:
and then change this to 
Referenced a ticket if any in the PR title or description